src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -529,7 +529,7 @@ def _activityMap(self, rseName):
         """
         if not self.isT0agent and not rseName.endswith("_Tape"):
             return "Production Output"
-        elif self.isT0agent and rseName.endswith("_Tape"):
+        elif self.isT0agent and "Tape" in rseName:
```

Comment:
I would suggest a check like:
``` 
elif self.isT0agent and (rseName.endswith("_Tape") or rseName.endswith("_Tape_Test"):
``` 

People are very creative and I wouldn't be surprised to, one day, see a Disk endpoint with `Tape` in its name :-D

Reply:
I can second [Alan's](@amaltaro) suggestion here.